VideoLocker/res/layout/row_course_list.xml

```diff
@@ -126,48 +108,18 @@
                 tools:targetApi="17" />
         </LinearLayout>
 
-        <LinearLayout
-            android:id="@+id/schoolcode_layout"
+        <org.edx.mobile.view.custom.ETextView
+            android:id="@+id/course_detail"
+            style="@style/course_card_course_run"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_alignParentLeft="true"
             android:layout_alignParentStart="true"
-            android:layout_alignTop="@id/starting_from"
             android:layout_below="@id/course_name"
-            android:layout_toLeftOf="@id/starting_from"
-            android:layout_toStartOf="@id/starting_from"
-            android:orientation="horizontal"
-            tools:targetApi="17">
-
-            <org.edx.mobile.view.custom.ETextView
-                android:id="@+id/org_code_tv"
-                style="@style/course_card_course_run"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="center_vertical"
-                tools:text="MITx"/>
-
-            <TextView
-                android:id="@+id/or_tv"
-                style="@style/course_card_course_run"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="center_vertical"
-                android:layout_marginLeft="3dp"
-                android:layout_marginRight="3dp"
-                android:layout_marginStart="3dp"
-                android:layout_marginEnd="3dp"
-                android:text="@string/or_symbol" />
-
-            <org.edx.mobile.view.custom.ETextView
-                android:id="@+id/school_code_tv"
-                style="@style/course_card_course_run"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="center_vertical"
-                android:ellipsize="end"
-                tools:text="6.002x" />
-        </LinearLayout>
+            android:layout_gravity="center_vertical"
+            android:layout_toRightOf="@+id/new_course_content_layout"
```

Comment:
That'll require nesting of multiple `LinearLayouts`. Look at my commit https://github.com/edx/edx-app-android/commit/28f7ae88de3a3daacf6ec92007f6969ec53b2d22.

I think its better if we use `RelativeLayout` here. It'll also turn out to be better if in future we are to add more overlapping views.